cmd/convox/Makefile

```diff
@@ -6,13 +6,16 @@ work   := $(gopath)/src/github.com/convox/convox
 all: build
 
 build:
-	mkdir pkg/
+	mkdir -p pkg/
 	export CGO_ENABLED=0
-	env GOOS=linux GOARCH=amd64 go build -ldflags "-X main.version=$(VERSION) -X main.image=$(IMAGE)" -o pkg/convox-linux-amd64
-	env GOOS=linux GOARCH=arm64 go build -ldflags "-X main.version=$(VERSION) -X main.image=$(IMAGE)" -o pkg/convox-linux-arm64
-	env GOOS=darwin GOARCH=amd64 go build -ldflags "-X main.version=$(VERSION) -X main.image=$(IMAGE)" -o pkg/convox-darwin-amd64
-	env GOOS=darwin GOARCH=arm64 go build -ldflags "-X main.version=$(VERSION) -X main.image=$(IMAGE)" -o pkg/convox-darwin-arm64
+	env GOOS=linux GOARCH=amd64 go build -tags=hidraw -ldflags "-X main.version=$(VERSION) -X main.image=$(IMAGE)" -o pkg/convox-linux-amd64
```

Comment:
This tag changes the changes the USB HID to use the `hidraw` backend for `hidapi`, instead of `libusb`. It'll enable Linux users to use the Yubikey.
See [go-u2fhost Linux compatibility](https://github.com/convox/go-u2fhost#linux).